src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveTestTest.template

```diff
@@ -0,0 +1,233 @@
+        {
+            public Vector<{MaskBaseType}> _fld1;
+
+            public static TestStruct Create()
+            {
+                var testStruct = new TestStruct();
+
+                testStruct._fld1 = {Op1Value};
+
+                return testStruct;
+            }
+
+            public void RunStructFldScenario(TestTest__{TestName} testClass)
+            {
+                Vector<{MaskBaseType}> op2 = {Op2Value};
+
+                var result = {Isa}.{Method}(_fld1, op2);
+
+                testClass.ValidateResult(_fld1, op2, result);
+            }
+        }   
+
+        private Vector<{MaskBaseType}> _fld1;
+
+        public TestTest__{TestName}()
+        {
+            if ({Isa}.IsSupported)
+            {
+                _fld1 = {Op1Value};
+            }
+            else
+            {
+                _fld1 = default;
+            }
+            Succeeded = true;
+        }
+
+        public bool IsSupported => {Isa}.IsSupported;
+
+        public bool Succeeded { get; set; }
+
+        public void RunBasicScenario()
+        {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunBasicScenario));
+
+            Vector<{MaskBaseType}> op1 = {Op1Value};
+            Vector<{MaskBaseType}> op2 = {Op2Value};
+
+            var result = {Isa}.{Method}(op1, op2);
+
+            ValidateResult(op1, op2, result);
+        }
+
+        public void RunReflectionScenario()
+        {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunReflectionScenario));
+
+            Vector<{MaskBaseType}> op1 = {Op1Value};
+            Vector<{MaskBaseType}> op2 = {Op2Value};
+
+            var result = typeof({Isa}).GetMethod(nameof({Isa}.{Method}), new Type[] { typeof(Vector<{MaskBaseType}>), typeof(Vector<{MaskBaseType}>) })
+                                     .Invoke(null, new object[] {
+                                        op1, op2
+                                     });
+
+            ValidateResult(op1, op2, (bool)result);
+        }
+
+        public void RunClassFldScenario()
+        {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunClassFldScenario));
+
+            Vector<{MaskBaseType}> op2 = {Op2Value};
+
+            var result = {Isa}.{Method}(_fld1, op2);
+
+            ValidateResult(_fld1, op2, result);
+        }
+
+        public void RunStructLclFldScenario()
+        {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunStructLclFldScenario));
+
+            Vector<{MaskBaseType}> op2 = {Op2Value};
+
+            var test = TestStruct.Create();
+            var result = {Isa}.{Method}(test._fld1, op2);
+
+            ValidateResult(test._fld1, op2, result);
+        }
+
+        public void RunStructFldScenario()
+        {
+            TestLibrary.TestFramework.BeginScenario(nameof(RunStructFldScenario));
+
+            var test = TestStruct.Create();
+            test.RunStructFldScenario(this);
+        }
+
+        public void RunUnsupportedScenario()
```

Comment:
same here. Can you please reuse the existing template? If not, can you confirm from which template this was created and what were the differences that was not letting us reuse it?

Reply:
The template was `ExtractVectorTest.template`. The only differences are alignment and when using the `Load` APIs, I needed to create a mask.